### PR TITLE
Clarified use of gob.Register()

### PR DESCRIPTION
**Pull request by crantok**
Proposing docs change after posting this support request:
https://groups.google.com/forum/#!topic/gorilla-web/cPcbiqFC3GU

### CODE REVIEW

**Comment by kisielk** [comment]
I would reword this slightly. "empty" means something particular in the Go spec. For example an empty struct is a struct with no fields, or an empty interface is an interface with no methods. I think the appropriate description would be zero value or nil pointer. However, there's no requirement that the value is zero or the pointer is nil when calling `gob.Register`. I think the documentation should just say "a value of the desired type"

**Comment by crantok** [comment]
That sounds good to me. I'll make that change.

**Comment by kisielk** [comment]
Thanks 👍

**Comment by elithrar** [comment]
Thanks @crantok!

**Comment by crantok** [comment]
My  pleasure :)
